pkg/types/contract_reader.go

```diff
@@ -17,6 +17,10 @@ const (
 	ErrNotFound                    = NotFoundError("not found")
 )
 
+type BlockMetaData struct {
+	Height uint64
+}
```

Comment:
we can add to this as needed, don't think we need to finalise on what data to expose at this point?